jerry-core/ecma/builtin-objects/ecma-builtin-global.cpp

```diff
@@ -125,7 +126,8 @@ ecma_builtin_global_object_parse_int (ecma_value_t this_arg __attr_unused___, /*
   ecma_length_t end = str_size;
   for (ecma_length_t i = 0; i < end; i++)
   {
-    if (!(isspace (utf8_string_buff[i])))
+    if (!lit_char_is_white_space (utf8_string_buff[i])
+        && !lit_char_is_line_terminator (utf8_string_buff[i]))
```

Comment:
@zherczeg, would it be ok, if I move the `isspace`-like functions to `ecma-builtins-json.cpp` (currently it is the only module that uses the functions), and raise an issue for replacing them with proper `lit_char_*` helpers?